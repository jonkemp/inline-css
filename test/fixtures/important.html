<html><link rel="stylesheet" href="important.css" /><body>
<a id="a" style="color: blue; background: green;">woot</a></body></html>
