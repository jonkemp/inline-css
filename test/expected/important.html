<html><head></head><body>
<a id="a" style="background: blue !important; border-style: dashed !important; color: red !important;">woot</a>
</body></html>
